components/ai_chat/renderer/page_text_distilling.cc

```diff
@@ -201,7 +201,7 @@ void DistillPageText(
     auto on_script_executed =
         [](base::OnceCallback<void(const std::optional<std::string>&)> callback,
            std::optional<base::Value> value, base::TimeTicks start_time) {
-          if (value->is_string()) {
+          if (value && value->is_string()) {
             std::move(callback).Run(value->GetString());
```

Comment:
The string can be quite big, and we can move it here. Let's just do this:
`std::move(*value)->TakeString()`